ProcessMaker/Repositories/CaseParticipatedRepository.php

```diff
@@ -54,32 +56,18 @@ public function create(CaseStarted $case, TokenInterface $token): void
     public function update(CaseStarted $case, TokenInterface $token)
     {
         try {
-            $caseParticipated = CaseParticipated::where('user_id', $token->user->id)
-                ->where('case_number', $case->case_number)
-                ->first();
+            if (!$this->checkIfCaseParticipatedExist($token->user->id, $case->case_number)) {
```

Comment:
What if the $token belongs to an self service assignment the user will be null

Reply:
Right, but the validation is done in the `CaseRepository>updateParticipants()` method